### DISCUSSION TITLE
5.0版本中二次封装组件时如何使用 CSSinJS

### DISCUSSION THREAD

**Discussion by liuweiGL** [ant-design/ant-design#40617] [Q&A] [answered]
在封装业务组件时我想使用 `genComponentStyleHook` 编写样式，但是这个方法目前是从 `antd/es/theme/internal` 导出的，请问是否可以直接 import 使用？

**Comment by liuweiGL** [comment]
或者业务项目中还是保持使用原先的 less 即可？

**Reply by qc-z** [reply]
二次封装的组件不知道如何写样式，调用方使用ConfigProvider无法覆盖自定义样式

**Reply by liuweiGL** [reply]
如果你还是封装成基础组件的话可以参考一下这个 https://github.com/formilyjs/antd

**Reply by qc-z** [reply]
非常感谢

**Comment by Wxh16144** [comment, marked as answer]
建议关注一下 #40489，这是 antd 正在讨论的一个最佳实践。
